text/0000-lts.md

```diff
@@ -0,0 +1,164 @@
+- Feature Name: lts
+- Start Date: 2018-06-21
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Add an additional release channel for long term releases.
+
+# Motivation
+[motivation]: #motivation
+
+Rust's current release model has, overall, worked very well for us. We maintain
+a strong degree of backwards compatibility despite producing a new stable
+release very frequently. Relative to many other languages, it is very easy to
+upgrade between Rust stable releases.
+
+However, for various institutional reasons, many users do not find it
+convenient and desirable to upgrade with the frequency that Rust releases. So
+far, they have been left on their own to figure out when to actually upgrade,
+creating a situation of uncertainty and disparity. Moreover, because there are
+no norms around when an older stable stops being "supported," libraries authors
+experience a similar level of uncertainty around which versions of the compiler
+their library should be compatible with.
+
+We solve this problem by adding long-term support ("LTS") release channels,
+providing explicit support for releases beyond the latest stable.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+## Two new channels: lts and lts-prior
```

Comment:
It's unclear to me why _two_ new channels are needed.  Could you elaborate on that in the RFC?